source/updateCheck.py

```diff
@@ -745,14 +753,15 @@ def initialize():
 			"lastCheck": 0,
 			"dontRemindVersion": None,
 			"pendingUpdateVersion": None,
-			"pendingUpdateVersionTuple": None,
+			"pendingUpdateAPIVersion": None,

```

Comment:
in setStateToNone, you're setting this to (0, 0, 0), whereas here, this is set to None. Would it help to make this consistent?